src/containers/AuthorList/index.tsx

```diff
@@ -3,7 +3,7 @@ import Author from '../../components/Author';
 import './AuthorList.scss';
 
 interface Props {
-  authors: string[];
+  authors?: string[];
```

Comment:
Why are you making authors optional?

Reply:
We are making this optional because there is a `null` conditional check already in place for `authors` but it's required in the `types`.

Moreover, there was already a test written for a `null` value of `authors` but was showing error in VSCode which turned out to be a genuine one because a `string[]` would never accept a `null`.

Not sure if this would ever be the case, but if we always expect `authors`, then I would suggest we remove the `null` check in the actual code and it's corresponding test as well. Otherwise, as per my limited knowledge, I think this should ideally be `optional`.

Please let me know if I misunderstood something. Thanks.

Reply:
An article will always have an author.

Reply:
Oh ok @benhalverson, we can remove the unnecessary check in the main component then. I'll remove it. Thanks for confirming!